build.gradle

```diff
@@ -21,7 +21,7 @@ buildscript {
 
 group = 'com.synopsys.integration'
 
-version = '9.5.0-SNAPSHOT'
+version = '9.5.0-SIGQA2-devm.IDETECT-3587-PI-SNAPSHOT'
```

Comment:
Should this be undone?

Reply:
Yes this will be changed once PI is released until then QA will resume testing this on feature branches.